backend/src/data/repositories/interview/interview.repository.ts

```diff
@@ -23,24 +23,22 @@ class Interview {
   public getAll(): Promise<InterviewsGetAllItemResponseDto[]> {
     return this.#InterviewModel
       .query()
-      .withGraphJoined('courseCategory')
-      .withGraphJoined('interviewee(withoutPassword).[userDetails]')
-      .withGraphJoined('interviewer(withoutPassword).[userDetails]')
+      .withGraphJoined(
+        '[courseCategory, interviewee.[userDetails], interviewer.[userDetails]]',
+      )
       .castTo<InterviewsGetAllItemResponseDto[]>()
       .execute();
   }
 
-  public async getById(id: number): Promise<InterviewsByIdResponseDto | null> {
-    const interview = await this.#InterviewModel
+  public getById(id: number): Promise<InterviewsByIdResponseDto | null> {
+    return this.#InterviewModel
```

Comment:
It'll return `InterviewsByIdResponseDto | undefined`, I think it should be as before

Reply:
@fshabanov when I wrote earlier db requests, @PavelSukhinin told me to use better .execute() than async await
@what1s1ove could you assist in this question please?

Reply:
you just need to add a check for null

```ts
const entity = await repos.getEntity(); // can return indefined

return entity ?? null // this check
```